packages/botonic-cli/src/commands/new.ts

```diff
@@ -113,7 +116,12 @@ Creating...
     spinner.succeed()
     await this.botonicApiService.buildIfChanged(false)
     this.botonicApiService.beforeExit()
-    await exec('mv ../.botonic.json .')
+    try {
+      await fsExtra.move('../.botonic.json', './botonic.json')
```

Comment:
Not sure if it's a typo or not, it's `.botonic.json` to `botonic.json` without dot at the beginning but in the original it just moves the file one directory up - is that what we want?